hack/client/generate-client.sh

```
@@ -0,0 +1,40 @@
+# The kube_codegen.sh is from 'https://raw.githubusercontent.com/kubernetes/code-generator/refs/tags/v0.30.0/kube_codegen.sh'.
+source hack/client/kube_codegen.sh
+
+BACKUP_DIR=$(mktemp -d)
+
+echo "Backup the original client directory to ${BACKUP_DIR}."
+
+# Backup the original client directory.
+mv $(pwd)/${OUTPUT_DIR} ${BACKUP_DIR}
+
+GOBIN=${GOBIN} kube::codegen::gen_client \
+  $(pwd) \
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)